charon/src/translate/translate_ctx.rs

```diff
@@ -441,12 +441,55 @@ impl<'tcx, 'ctx> TranslateCtx<'tcx, 'ctx> {
         let opaque = attributes
             .iter()
             .any(|attr| attr == "charon::opaque" || attr == "aeneas::opaque");
+        let rename = {
+            let str = attributes.iter().find(|str| {
+                str.starts_with("charon::rename(") || str.starts_with("aeneas::rename(")
```

Comment:
`starts_with` is redundant with the `strip_prefix` that follows

Reply:
To make this code more robust, it would also be nice to have a `parse_attribute(&str) -> Option<Attribute>` function, with
```rust
enum Attribute {
  Opaque,
  Rename(String),
}
```